database/staking.go

```diff
@@ -99,15 +117,62 @@ func (db BigDipperDb) SaveValidators(validators []bstaking.Validator) error {
 	return err
 }
 
-// SaveValidatorUptime stores into the database the given validator uptime information.
-// It assumes that for each uptime information provided, the associated validator data
-// have already been saved inside the database properly.
-func (db BigDipperDb) SaveValidatorUptime(uptime bstaking.ValidatorUptime) error {
-	statement := `INSERT INTO validator_uptime (height, validator_address, signed_blocks_window, missed_blocks_counter)
-				  VALUES ($1, $2, $3, $4)`
-	_, err := db.Sql.Exec(statement,
-		uptime.Height, uptime.ValidatorAddress.String(), uptime.SignedBlocksWindow, uptime.MissedBlocksCounter)
-	return err
+func (db BigDipperDb) SaveEditValidator(validator sdk.ValAddress, commissionRate int64, minSelfDelegation int64,
```

Comment:
As far as I've seen, this method performs the following operations: 

- check if the validator exists
- get the latest validator info and commission
- edit the validator info and commission

I personally have some suggestions on how the implementation of this method might be made more easier to understand: 

1. Split this method into smaller methods.
   - To get the latest validator info create a new `GetValidatorInfo` method that accepts the consensus address and returns the validator info data
   - To get the latest validator commission create a new `GetValidatorCommission` method that accepts the consensus address and returns the latest commission rate for such validator
   - To store the validator info, create a new `SaveValidatorInfo` method 
   - To store the validator commission, create a new `SaveValidatorCommission` method 

2. Create specific types for each table.  
   Since we are dealing with a lot of tables, it would be much easier to have custom types for each of them in order to represent a single entry inside them.  
   An example could be:  
   ```golang
   type ValidatorCommission struct {
     ValidatorAddress string `db:"validator_address"`
     Timestamp time.Time `db:"timestamp"`
     Commission float64 `db"commission"`
     MinSelfDelegation int64 `db:"min_self_delegation"`
     Height int64 `db:"height"`
   }
   ```
   This would make it much easier for you to both insert and query data using [`Sqlx` support to query and insert data using custom struct definitions](https://stackoverflow.com/a/40884611/5162161) ( an example inside BDJuno can be found [here](https://github.com/forbole/bdjuno/blob/master/database/staking.go#L30)).